app/Controllers/importExportController.php

```diff
@@ -720,76 +720,6 @@ private function addFeedJson($origin) {
 		return $return;
```

Comment:
The diff concerning this file is a bit complicated to review, I suggest to only read the new file